addons/prometheus/template/base/operator/matchlabel-instance.yaml

```diff
@@ -0,0 +1,10 @@
+- op: remove
+  path: /spec/selector/matchLabels/app.kubernetes.io~1instance
+- op: remove
+  path: /spec/selector/matchLabels/app.kubernetes.io~1name
```

Comment:
is this supposed to be like that ?  I mean, wasn't it supposed to be `app.kubernetes.io/instance` and `app.kubernetes.io/name` ?

Reply:
You can't have a `/` in the path - you need to escape it. `~1` is escaped `/`, `~0` is escaped `~`.

Reply:
From jsonpatch.com,

```
If you need to refer to a key with ~ or / in its name, you must escape the characters with ~0 and ~1 respectively. For example, to get "baz" from { "foo/bar~": "baz" } you’d use the pointer /foo~1bar~0.
```

Reply:
Very intuitive. OK, lesson learned, thanks for clarifying.